app/src/main/java/com/devshish/internship/domain/repository/IProfileRepository.kt

```diff
@@ -1,11 +1,8 @@
 package com.devshish.internship.domain.repository
 
-import com.devshish.internship.domain.model.User
-import kotlinx.coroutines.flow.Flow
+import com.devshish.internship.data.model.UserInfoDTO
 
 interface IProfileRepository {
 
-    fun getUser(): Flow<User>
-
-    suspend fun editUser(updatedUser: User)
+    suspend fun getUser(): UserInfoDTO
```

Comment:
It seems like you do not understand completely idea of separating models on different layers - we will talk through it later (just ask me on Monday, if I not forget)

1. You are already have User model - why not use it? (And change it a little bit for our needing)
2. You use network model UserInfoDTO right on domain, instead of using just mapping it inside of repository